bin/codecov.sh

```diff
@@ -41,14 +41,14 @@ COVERAGEDIR="$(mktemp -d /tmp/istio_coverage.XXXXXXXXXX)"
 mkdir -p "$COVERAGEDIR"
 
 function cleanup() {
-  make localTestEnvCleanup
+  make -f Makefile.core.mk localTestEnvCleanup
```

Comment:
Why do we have to the `-f`, is it so we don't do docker-in-docker if we are using container?

Reply:
Correct.